dual_net.py

```diff
@@ -472,15 +482,15 @@ def validate(tf_records, validate_name=None):
         def input_fn(params):
             return preprocessing.get_tpu_input_tensors(
                 params['batch_size'],
-                tf_records)
+                tf_records, filter_amount=0.05)
```

Comment:
because the holdout games aren't pre-chunked or otherwise randomized; we'd dramatically misapproximate our value error by using all the positions from each of the holdout games